extensions/amp-story/1.0/amp-story-interactive-binary-poll.js

```diff
@@ -187,6 +195,23 @@ export class AmpStoryInteractiveBinaryPoll extends AmpStoryInteractive {
     return convertedOption;
   }
 
+  /**
+   * Creates a number to transfrom the x axis of binary poll text.
+   * @param {number} percentage
+   * @return {number}
+   * @private
+   */
+  getTransformVal_(percentage) {
+    let mappedVal = percentage - 50;
+    if (mappedVal < MIN_HORIZONTAL_TRANSFORM) {
+      mappedVal = MIN_HORIZONTAL_TRANSFORM;
+    }
```

Comment:
let mappedVal = Math.max(percentage - 50, MIN_HORIZONTAL_TRANSFORM);

Sorry I didn't suggest this before, I think the other commits had a - that I didn't know if it was a bug or intentional, but now you can replace the 4 lines with the line above, which is easy to read and smaller 😁

Reply:
🔥 Thank you Matias! Done.